docs/source/topics/proc_installing-codeready-containers.adoc

```diff
@@ -1,11 +1,57 @@
 [id="installing-codeready-containers_{context}"]
 = Installing {prod}
 
+{prod} is available as a portable executable for {rhel} and {msw}.
```

Comment:
I'm not used to seeing `executable` as a noun. But your users might be, in which case, keep it!

Reply:
This has been discussed before, actually. :)

Normally, I write "binary," but it was brought to my attention before that technically, we're discussing an _executable_ file, not just a binary one. It's a subtle difference, but an important one. :)